Example/Example/Purchase/PurchaseLogicController.swift

```diff
@@ -194,13 +184,28 @@ final class PurchaseLogicController {
     }
   }
 
-  func loadCashAppToken() {
+  func retrieveCashAppToken(cashButton: CashAppPayButton? = nil) {
+    if cashButton != nil {
+      self.cashButton = cashButton
+    }
+    isCashButtonEnabled(false)
```

Comment:
This function is incorrectly named -- it should be called `setCashButtonEnabled(_ enabled: Bool)` -- but you could get rid of this function entirely and just rewrite these lines as 

```
if let cashButton = cashButton {
    self.cashButton = cashButton
}
self.cashButton?.isEnabled = false
```

Reply:
renamed in [b7f8283](https://github.com/afterpay/sdk-ios/pull/243/commits/b7f8283d641dcf9495116ca216815db7299687af)